tests/storage/test_data_import_cron.py

```diff
@@ -205,6 +208,63 @@ def second_object_cleanup(
     resource_class(namespace=namespace.name, name=second_object_name).clean_up()
 
 
+@pytest.fixture()
+def data_import_cron_pvc_source(
+    unprivileged_client,
+    data_import_cron_namespace,
+    storage_class_with_filesystem_volume_mode,
+    pvc_for_source_data_import_cron,
+):
+    # cron_template_spec = golden_images_data_import_cron_spec.template.spec
+    with DataImportCron(
+        name="pvc-import-cron",
+        namespace=data_import_cron_namespace.name,
+        schedule="*/1 * * * *",
+        managed_data_source="pvc-import-datasource",
+        annotations=BIND_IMMEDIATE_ANNOTATION,
+        template={
+            "spec": {
+                "source": {
+                    "pvc": {
+                        "name": pvc_for_source_data_import_cron.name,
+                        "namespace": pvc_for_source_data_import_cron.namespace,
+                    }
+                },
+                "sourceFormat": "pvc",
+                "volumeMode": "Filesystem",
+                "storage": {
+                    "storageClassName": storage_class_with_filesystem_volume_mode,
+                    "accessModes": [PersistentVolumeClaim.AccessMode.RWO],
+                    "resources": {"requests": {"storage": "10Gi"}},
+                },
+            }
```

Comment:
Are you sure we need these?
```
"volumeMode": "Filesystem",
"storageClassName": storage_class_with_filesystem_volume_mode,
"accessModes": [PersistentVolumeClaim.AccessMode.RWO],
```
I see that we are not specifying these in the default  DataImportCrons:

```
$ oc get DataImportCron -n openshift-virtualization-os-images fedora-image-cron -oyaml
apiVersion: cdi.kubevirt.io/v1beta1
kind: DataImportCron
...
spec:
  garbageCollect: Outdated
  managedDataSource: fedora
  schedule: 3 9/12 * * *
  template:
    metadata: {}
    spec:
      source:
        registry:
          pullMethod: node
          url: docker://quay.io/containerdisks/fedora:latest
      storage:
        resources:
          requests:
            storage: 30Gi
    status: {}

```